### PR TITLE
feat (contrib/actions): add cds-nexus-upload action

### PR DESCRIPTION
**Pull request by bnjjj**

### CODE REVIEW

File: contrib/actions/cds-nexus-upload.hcl [hunk 1]
```diff
@@ -0,0 +1,63 @@
+
+name = "CDS_NexusUpload"
+description = "Upload file on Nexus"
+
+// Requirements
+requirements = {
```
Comment (yesnault) [inline contrib/actions/cds-nexus-upload.hcl]: :%s/requirements/parameters

Comment (yesnault) [inline contrib/actions/cds-nexus-upload.hcl]: description attr on all fields:

```hcl
"url" = {
		type = "string"
		description = "git URL"
		value = "{{.cds.app.repo}}"
	}
```

File: contrib/actions/cds-nexus-upload.hcl [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+	"file" = {
+		type = "string"
+	}
+	"packaging" = {
+		type = "string"
+	}
+}
+
+// Parameters
+parameters = {
```
Comment (yesnault) [inline contrib/actions/cds-nexus-upload.hcl]: requirement -> binary curl

File: contrib/actions/cds-nexus-upload.hcl [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	"artifactId" = {
+		type = "string"
+		value = "{{.cds.application}}"
+	}
+	"version" = {
+		type = "string"
+		value = "{{.cds.build.VERSION}}"
+	}
+	"file" = {
```
Comment (sguiheux) [inline contrib/actions/cds-nexus-upload.hcl]: file -> files to allow regexp

File: contrib/actions/cds-nexus-upload.hcl [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+// Steps
+steps = [{
+	script = <<EOF
+#!/bin/bash
+set -e
+
+echo "action git from directory"
+pwd
+echo "Upload to Nexus ({{.url}}) on repository {{.repository}}"
+curl -F r={{.repository}} -F hasPom=false -F e={{.extension}} -F g="{{.groupId}}" -F a="{{.artifactId}}" -F v="{{.version}}" -F p={{.packaging}} -F file={{.file}} -u {{.login}}:{{.password}} {{.url}}
```
Comment (sguiheux) [inline contrib/actions/cds-nexus-upload.hcl]: loop on files

File: contrib/actions/cds-nexus-upload.hcl [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+description = "Upload file on Nexus"
+
+// Requirements
+requirements = {
+	"bash" = {
+		type = "binary"
+		value = "bash"
+	}
+	"login" = {
+		type = "string"
```
Comment (fsamin) [inline contrib/actions/cds-nexus-upload.hcl]: set a default value for login parameter (which should be in parameters section not in requirements) = {{.cds.proj.nexus.login}}

File: contrib/actions/cds-nexus-upload.hcl [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	"password" = {
+		type = "string"
```
Comment (fsamin) [inline contrib/actions/cds-nexus-upload.hcl]: set a default value for password parameter (which should be in parameters section not in requirements) = {{.cds.proj.nexus.password}}

File: contrib/actions/cds-nexus-upload.hcl [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+	}
+	"url" = {
+		type = "string"
+		description = "Nexus URL"
```
Comment (fsamin) [inline contrib/actions/cds-nexus-upload.hcl]: set a default value for url parameter (which should be in parameters section not in requirements) = {{.cds.proj.nexus.url}}

File: contrib/actions/cds-nexus-upload.hcl [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+	
+}
+
+// Steps
+steps = [{
+	script = <<EOF
+#!/bin/bash
+set -e
+
+echo "action git from directory"
```
Comment (fsamin) [inline contrib/actions/cds-nexus-upload.hcl]: remove or fix this `echo`

File: contrib/actions/cds-nexus-upload.hcl (hunk 4 above)
Comment (fsamin) [inline contrib/actions/cds-nexus-upload.hcl]: maybe you should explicitly check if file exists

File: contrib/actions/cds-nexus-upload.hcl [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+		value = "{{.cds.application}}"
+	}
+	"artifactId" = {
+		type = "string"
+		description = "Artifact id of the artifact"
+		value = "{{.cds.application}}"
+	}
+	"version" = {
+		type = "string"
+		description = "Version of the artifact. Supports resolving of "LATEST", "RELEASE" and snapshot versions ("1.0-SNAPSHOT") too."
```
Comment (yesnault) [inline contrib/actions/cds-nexus-upload.hcl]: to many double quote

Comment (bnjjj) [inline contrib/actions/cds-nexus-upload.hcl]: Oups, my bad, missing syntax highlighter

Done.

**Pull request merged**